Checkmarx/sast-to-ast-export
# AST-21193

By submitting a PR to this repository, you agree to the terms within the [Checkmarx Code of Conduct](../CODE-OF-CONDUCT.md). Please see the [contributing guidelines](../CONTRIBUTING.md) for how to create and submit a high-quality PR for this repo.

### Description

- Added new filter presets, to only include the ones associated to projects.
- When Project IDs is not defined exports all presets.

### References

AST-21193

### Testing

To test :

- Go to SAST create one custom Preset, associated to one project and run CLI passing in args that project ID, should return only the presets associated to projects (excluding all base/default for now).
- Run CLI without project IDs args and should return all presets (excluding all base/default for now)

### Checklist
